routes/api/share.js

```diff
@@ -0,0 +1,64 @@
+import Student from "../../models/Student";
+import express from 'express';
+import TimeTable from "../../models/TimeTable";
+import mongoose from 'mongoose';
+//import { check, validationResult } from "express-validator";
+const router = express.Router();
+
+
+let ObjectId = mongoose.Types.ObjectId; 
+router.get("/shareTT",async (req, res) => {
+    let branch = req.query.branch;
+    let year = req.query.year;
+    if(year === undefined || isNaN(year))
+    {
+       
+        res.send(null);
+    }
```

Comment:
Both work, but using `express-validator` for this checking would be a neater option I think.

Reply:
I wasn't really sure on how to check for NaN and how a get request would treat it

Reply:
Year is a string, it will be same snippet as `Helform.js`